src/Core/Framework/Plugin/PluginService.php

```diff
@@ -100,9 +100,18 @@ public function refreshPlugins(Context $shopwareContext, IOInterface $composerIO
             $authors = null;
```

Comment:
would you mind extracting the whole author logic into an own private method?

Reply:
Done